### ISSUE TITLE
Add section "Help for Screen Readers"

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#511]
Include use of CASE issue in Names.

**Comment by PatReynoldsFUG** [comment]
Added to Draft Accessibility Statement.  May not be 
a) correct or
b) the whole story.

Could you take a look @AlOneill ?
Thanks, Pat

**Comment by AlOneill** [comment]
Now that we know how to get ARIA attributes published on Refinery pages, I suggest that we can remove the information currently under the heading "Advice for users of screen reading software".

Does "Include use of CASE issue in Names." need addressing? If yes, then how?

**Comment by PatReynoldsFUG** [comment]
"Include use of CASE issue in Names." means advice should be provided on how we use case to distinguish between given names and family names in help.  Or we should perhaps be providing a separate way for a screen reader user to be informed that Susan Bowes are given names and Lyon is a family name in one example and that  that Susan is given names and Bowes Lyon is a family name in another example.   Maybe not use casing at all but write the examples out?

**Comment by AlOneill** [comment]
I should add that my suggestion for removing the information under the heading "Advice for users of screen reading software" depends on the Refinery configuration being updated on live sites as well as test sites.

Telling screen reader users _how_ we use letter casing will not do much in itself because the case is hidden from those users — letters, words and syllables sound the same irrespective of the case used. We would have to be explicit. I think that is that what is meant by "write the examples out".

We could keep the casing on the grounds that that is what many researchers expect, but we must not fool ourselves that letter casing works for all our users. And the Results are not consistent in the use of upper case because the G.R.O. are not consistent and transcribers have copied as is.

**Comment by PatReynoldsFUG** [comment]
Agreed, @AlOneill

**Comment by DeniseColbert** [comment]
Will be superseded by improvements to Names: closing
